### ISSUE TITLE
Key error

### ISSUE THREAD

**Issue by fernandojunior** [pseudonym117/Riot-Watcher#28]
Get a summoner with special char results a Key Error...

```py
from riotwatcher import RiotWatcher, BRAZIL
w = RiotWatcher(key='<key>', default_region=BRAZIL)
me = w.get_summoner(name='faucheurdecrân')
# KeyError: 'faucheurdecr\xc3\xa2n'
```

So, I recommend to use this strategy in RiotWatcher#get_summoner method:
```py
return self.get_summoners(names=[name, ], region=region).itervalues().next()
```
instead of 
```py
return self.get_summoners(names=[name, ], region=region)[name]
```
Final method:
```py
    def get_summoner(self, name=None, _id=None, region=None):
        if (name is None) != (_id is None):
            if name is not None:
                # name = self.sanitized_name(name)  redundant statement ... it is called in RiotWatcher#get_summoners method
                return self.get_summoners(names=[name, ], region=region).itervalues().next()
            else:
                return self.get_summoners(ids=[_id, ], region=region)[str(_id)]
        return None
```

**Comment by DeepBlaze** [comment]
Had the same problem!, for python 3.4 i used 

```python
return next (iter(self.get_summoners(names=[name, ], region=region).values()))
```

instead of
```python
return self.get_summoners(names=[name, ], region=region)[name]
```

**Comment by pseudonym117** [comment]
finally fixed this as of 190751a
